### ISSUE TITLE
Add LICENSE

### ISSUE THREAD

**Issue by astrofrog** [dotastro/hacks-collector#71]
As @bsipocz pointed out, we are missing a license... Any preference? We could just go with BSD?

**Comment by eteq** [comment]
Good point!  Should we just take the one from https://github.com/dotastro/hack-list-submission-app/blob/master/LICENSE ?  It has my name, though - should I change it to "the dotAstronomy team" or similar?
